xwiki-platform-core/xwiki-platform-web/xwiki-platform-web-war/src/main/webapp/resources/js/xwiki/usersandgroups/usersandgroups.js

```diff
@@ -55,6 +55,13 @@ window.MSCheckbox = Class.create({
     ];
     this.labels = ['','',''];
 
+    // Alts are the messages describing the checkbox images
+    this.alts = [
+      '$escapetool.javascript($services.localization.render("none"))',
+      '$escapetool.javascript($services.localization.render("allow_1"))',
+      '$escapetool.javascript($services.localization.render("allow_0"))'
```

Comment:
the keys are missing some namespacing. See https://dev.xwiki.org/xwiki/bin/view/Community/DevelopmentPractices#HTranslationBestPractices

Reply:
I decided to use the 'platform.core.rightsManagement.' prefix/namespace. 
I'm not sure I understood how I'm supposed to extend the translations.
Of what I understood, since these string are used in the core project, all I need to do is add them directly in the ApplicationResources.properties file. There is nothing to do with L10N.

`platform.core.rightsManagement.allowed=Allowed
platform.core.rightsManagement.denied=Denied
platform.core.rightsManagement.inherit=Inherit`

Reply:
You don't need to do anything for Weblate, that's automatic.